src/BIMDataComponents/BIMDataSafeZoneInline/BIMDataSafeZoneInline.vue

```diff
@@ -0,0 +1,54 @@
+<template>
+  <div class="safe-zone-inline">
+    <BIMDataButton
+      class="safe-zone-inline__btn-delete"
+      :style="{ order: reverse ? 2 : 0 }"
+      color="high"
+      fill
+      radius
+      @click="$emit('confirm-delete')"
+    >
+      {{ deleteTrad[locale] }}
+    </BIMDataButton>
+    <BIMDataButton
+      class="safe-zone-inline__btn-close"
+      :style="{ minWidth: 0, width: '15px', order: reverse ? 1 : 0 }"
```

Comment:
`order: reverse ? -1 : 0` may allow to remove the `order` on line 5 doesn't it ?